api/testing/testing_test.go

```diff
@@ -63,6 +63,7 @@ func TestAll(t *testing.T) {
 
 	server.StartPluginAPI(
 		nfs.Name,
+		"localhost:9100",
```

Comment:
is an sdk server running at this port? where was it created?

Reply:
No there isn't, will just set to the empty string to avoid confusion